### DISCUSSION TITLE
Do we have a way to preset default for (or) only one model?

### DISCUSSION THREAD

**Discussion by oliverbob** [open-webui/open-webui#1040] [General]
How do I set groq's mixtral 7b as the default model without clicking "set default"? I guess I have asked this somewhere when this project was young, but, couldn't find that thread.

If I can rephrase my question, how can I set a default model that persist throughout all future users (unless optionally they would like to change it)?

Thanks.

**Comment by flefevre** [comment]
I have eaxctly the same question: how can we set tge default models when the user connect to open webui?
I think it could be in config.json but i do not know the synthax

`{
	"version": 0,
	"ui": {
		"default_locale": "fr-FR",
		"prompt_suggestions": [
			{
				"title": ["Le CEA", "nouvelles sur le cea"],
				"content": "extraire sous la forme d'un tableau les informations contenues sur la page suivante #https://www.cea.fr/_layouts/15/i2i/web/ceasrchrss.ashx?pid=3748&wid=g_9cac7691_a0b2_4e18_850e_0aeff2541696 avec les colonnes titre, résumé et lien"
			},
			{
				"title": ["Show me a code snippet", "of a website's sticky header"],
				"content": "Show me a code snippet of a website's sticky header in CSS and JavaScript."
			}
		]
	}
}`
